### ISSUE TITLE
Unable to query Escape From Tarkov graphic settings

### ISSUE THREAD

**Issue by Gotchak** [RatScanner/RatScanner#485]
**Error**
Dll was not found.
```
System.DllNotFoundException: Dll was not found.
   at RatScanner.RatConfig.GetDpiForMonitor(IntPtr hmonitor, DpiType dpiType, UInt32& dpiX, UInt32& dpiY)
   at RatScanner.RatConfig.GetScalingForScreen(Screen screen)
   at RatScanner.RatConfig.GetTarkovScreenConfig()
   at RatScanner.RatConfig.TrySetScreenConfig()
   at RatScanner.RatConfig.LoadConfig()
   at RatScanner.View.BlazorUI..ctor()
   at RatScanner.PageSwitcher..ctor()
```
<details>
<summary>Log</summary>

```
[16:50:42.9076066] > [Warning] Config version (-1) is not supported!
[16:50:42.9421110] > [Warning] Unable to query Escape From Tarkov graphic settings.
[16:50:42.9471116] > System.DllNotFoundException: Dll was not found.
   at RatScanner.RatConfig.GetDpiForMonitor(IntPtr hmonitor, DpiType dpiType, UInt32& dpiX, UInt32& dpiY)
   at RatScanner.RatConfig.GetScalingForScreen(Screen screen)
   at RatScanner.RatConfig.GetTarkovScreenConfig()
[16:50:42.9481118] > [Error] Dll was not found.
 -------------------- 
 System.DllNotFoundException: Dll was not found.
   at RatScanner.RatConfig.GetDpiForMonitor(IntPtr hmonitor, DpiType dpiType, UInt32& dpiX, UInt32& dpiY)
   at RatScanner.RatConfig.GetScalingForScreen(Screen screen)
   at RatScanner.RatConfig.GetTarkovScreenConfig()
   at RatScanner.RatConfig.TrySetScreenConfig()
   at RatScanner.RatConfig.LoadConfig()
   at RatScanner.View.BlazorUI..ctor()
   at RatScanner.PageSwitcher..ctor()
[16:52:07.9544062] > [Warning] Config version (-1) is not supported!
[16:52:07.9879104] > [Warning] Unable to query Escape From Tarkov graphic settings.
[16:52:07.9929111] > System.DllNotFoundException: Dll was not found.
   at RatScanner.RatConfig.GetDpiForMonitor(IntPtr hmonitor, DpiTy

**Comment by Gotchak** [comment]
could be a WIN 7 Issue

**Comment by Blightbuster** [comment]
Yep its missing `Shcore.dll` which is only available on Windows 8.1 and later.
I only want to support Win10 anything below is out of scope sorry.
To fix it, you can try to download `Shcore.dll` and put it in the same folder as `RatScanner.exe` but no promises.
Thanks for reporting the issue never the less ❤️
